dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -1231,3 +1231,71 @@ def test_profile_merge_bin_edges_indices(self):
         profile_2.update(data_2)
 
         profile_1 + profile_2
+
+    def test_invalid_values_skew_kurt(self):
+        data = pd.Series(['-inf', 'inf'])
+        profiler = FloatColumn(data.name)
+
+        with warnings.catch_warnings(record=True) as w:
+            warnings.simplefilter("always")
```

Comment:
instead of using the warnings catch, if we want to simplify, we can utilize the `self.assertWarnsRegex`

Reply:
However, this seems to check for two warnings. we might have to do one at a time to validate, unsure.

Reply:
I'm curious if there's an obvious way to make it only return 1 warning for all 3 when it is an incoming data issue

Reply:
An idea can be that rather than checking var/skew/kurt individually, you can check if the sum is indeterminate/infinite on the values and warn there. Thoughts @JGSweets

Reply:
Makes sense to me.